packages/core/src/transport/httpRequest.spec.ts

```diff
@@ -62,44 +68,19 @@ describe('httpRequest', () => {
     expect(navigator.sendBeacon).toHaveBeenCalled()
     expect(server.requests.length).toEqual(1)
   })
-})
-
-describe('httpRequest parameters', () => {
-  const BATCH_BYTES_LIMIT = 100
-  let server: sinon.SinonFakeServer
-  let sendBeaconSpy: jasmine.Spy<(url: string, data?: BodyInit | null) => boolean>
-  beforeEach(() => {
-    server = sinon.fakeServer.create()
-    sendBeaconSpy = jasmine.createSpy()
-    navigator.sendBeacon = sendBeaconSpy
-  })
-
-  afterEach(() => {
-    server.restore()
-  })
 
-  it('should add batch_time', () => {
```

Comment:
Maybe we could keep a test on `batch_time`?

Reply:
I moved the batch_time logic as suggested by @bcaudan since it's only used for rum endpoints. I added tests here 
https://github.com/DataDog/browser-sdk/blob/15a5a4def8602fdeee8a102ab226069c6cba1c2c/packages/core/src/domain/configuration/transportConfiguration.spec.ts#L58-L67